content/js/query.md

```diff
@@ -54,18 +37,16 @@ let shareQuery = new skygear.Query(Note);
 shareQuery.greaterThan('share', 10);
 let query = skygear.Query.or(likeQuery, shareQuery);
 query.equalTo('category', 'diary');
-skygear.publicDB.query(query).then((notes) => {
-  console.log('Received notes with (like > 50 || share > 10) && category == diary');
-}, (error) => {
-  console.error(error);
-});
+skygear.publicDB.query(query);
+// (like > 50 || share > 10) && category == 'diary'
```

Comment:
- you can add "SQL equivalent:" at the beginning of the comment
- SQL use `OR` and `AND` instead of `||` and `&&`, also `=` not `==`